components/contributors/InfoCard.tsx

```diff
@@ -154,10 +162,9 @@ export default function InfoCard({
                   target="_blank"
                   rel="noreferrer"
                 >
-                  <div className="flex items-center text-secondary-500 hover:text-primary-300">
+                  <div className="flex items-center text-gray-500 hover:text-primary-300">
```

Comment:
why was all secondary changed to gray even with places without conflicts?

Reply:
I think i make some mistakes so, I will resolve ASAP